src/from-attribute.js

```diff
@@ -104,11 +106,62 @@ canReflect.assignSymbols(ElementAttributeObservable.prototype, {
 });
 
 module.exports = function fromAttribute (propertyName) {
-	return function fromAttributeBind (instance) {
-		return new Bind({
-			parent: new ElementAttributeObservable(instance, propertyName, {}, 'attributes'),
-			child: value.to(instance, propertyName),
-			queue: "domUI"
-		});
+	// Handle the class constructor
+	return function (ctr) {
+		if (ctr[metaSymbol] === undefined) {
+			ctr[metaSymbol] = {};
+		}
+		// Create array for all bindings that will be used within attributeChangedCallback
+		if (ctr[metaSymbol]._attributeBindings === undefined) {
+			ctr[metaSymbol]._attributeBindings = {};
+		}
+
+		// Modify the class prototype here
+		if (!ctr[metaSymbol]._hasInitializedAttributeBindings) {
+			// Set up the static getter for `observedAttributes`			
+			Object.defineProperty(ctr, "observedAttributes", {
+					get() {
+						return ctr[metaSymbol]._bindingAttributes;
+					}
+			});
+			
+			// Create the attributeChangedCallback handler
+			ctr.prototype.attributeChangedCallback = function (prop, oldVal, newVal) {
+					// Update property
+					if (prop === propertyName && newVal !== oldVal) {
```

Comment:
I don't think this will work for more than one attribute binding since this is only going to callback the handler for the last `propertyName` set up. `ctr.prototype.attributeChangedCallback` probably needs to also call the previous `attributeChangedCallback`.